webapp/packages/plugin-connection-search/src/Search/SearchDatabase.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+
+import { useService } from '@cloudbeaver/core-di';
+import {  PROJECT_GLOBAL_ID } from '@cloudbeaver/core-projects';
+import type { AdminConnectionSearchInfo } from '@cloudbeaver/core-sdk';
+import { ConnectionForm } from '@cloudbeaver/plugin-connections';
+
+import { ConnectionSearchService } from './ConnectionSearchService';
+import { DatabaseList } from './DatabaseList';
+
+export const SearchDatabase: React.FC = observer(function SearchDatabase() {
+  const connectionSearchService = useService(ConnectionSearchService);
+
+  function select(database: AdminConnectionSearchInfo) {
+    connectionSearchService.select(PROJECT_GLOBAL_ID, database);
+  }
```

Comment:
please use `activeProject` from `ProjectsService` (you can do it in `connectionSearchService.select`)